### PR TITLE
chore: Add documentation for openedX

### PR DESCRIPTION
**Pull request by Tj-Tracy**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -38,7 +38,17 @@ The page is currently hosted on http://localhost:1990/
...
+Currently, this MFE is not intergrated into the devstack, and must be run locally. This MFE requires credentials to be running, and will use a REST API from the Credentials IDA located at `credentials/apps/records/rest_api`.
+
+Credentials uses 2 enviroment variables to link to this MFE:
+- USE_LEARNER_RECORD_MFE -- Toggles the navigation in credentials to redirect to this MFE
+- LEARNER_RECORD_MFE_RECORDS_PAGE_URL -- The URL for the base URL of this MFE
+More details for these flags can be found in the base configuration of credentials :`credentials/settings/base`
+
+This MFE has 2 flags of its own:
+- SUPPORT_URL_LEARNER_RECORDS -- A link to a help/support center for learners who run into problems whilst trying to share their records
+- USE_LR_MFE -- A toggle that when on, uses the MFE to host shared records instead of the the old UI inside of credentials
```
Comment (MaxFrank13) [inline README.rst:50]: I think the `-` may need a space in front of them to make the bullet?

Screen Shot 2022-10-12 at 1 36 27 PM

File: README.rst [hunk 2]
```diff
@@ -24,12 +24,12 @@ Installation
 
 3. Install dependencies
 
-    .. code-block::
+   .. code-block::
       npm install
```
Comment (MaxFrank13) [inline README.rst:28]: Is this supposed to be visible in the normal view or just in an IDE?
Screen Shot 2022-10-12 at 1 40 52 PM

File: README.rst [hunk 3]
```diff
@@ -24,12 +24,12 @@ Installation
...
 
 4. Start the Learner Record MFE
 
-    .. code-block::
+   .. code-block::
       npm start
```
Comment (MaxFrank13) [inline README.rst:33]: Same question as above

File: README.rst (hunk 2 above)
Comment (Tj-Tracy) [reply README.rst:28]: Oh thats very weird I think my editor formatted the code blocks out of existence

**Review by MaxFrank13 — APPROVED** [review]
LGTM!

**Pull request merged**